include/deal.II/numerics/vector_tools_interpolate.templates.h

```diff
@@ -618,11 +618,30 @@ namespace VectorTools
   }
 
 
+
   template <int dim, int spacedim, typename VectorType>
   void
   get_position_vector(const DoFHandler<dim, spacedim> &dh,
                       VectorType &                     vector,
                       const ComponentMask &            mask)
+  {
+    const FiniteElement<dim, spacedim> &fe = dh.get_fe();
+    get_position_vector(
+      *fe.reference_cell().template get_default_mapping<dim, spacedim>(
+        fe.degree),
```

Comment:
> Not related to your patch, but is the "then a default one is used" wrong in line 274 and supposed to say "then a default zero is used"?

@bangerth These are the lines the comment is referring to.

Reply:
Oh, I get it now. "A default one" isn't meant to reads as "A default 1" but as "A default component mask". I'll fix that separately.